### PR TITLE
[BP-2260] Change kernel name to <CLUSTER_NAME>_<first five characters of CLUSTER_UUID>

### PR DESCRIPTION
**Pull request by bodo-sai**
Related PRs:
https://github.com/Bodo-inc/bodo-platform-jupyterlab-extension/pull/43
https://github.com/Bodo-inc/bodo-platform-backend/pull/669

### CODE REVIEW

File: remote_ikernel/manage.py [hunk 1]
```diff
@@ -191,66 +193,66 @@ def add_kernel(
     user.
     """
     kernel_name = []
-    display_name = []
+    display_name = cluster_name + "_" + cluster_uuid[:5]
```
Comment (sahil1105) [inline remote_ikernel/manage.py:196]:
```suggestion
    display_name =  f"{cluster_name} (ID: {cluster_uuid})"
```
I assume it'll automatically get truncated if it's too long?

File: remote_ikernel/manage.py [hunk 2]
```diff
@@ -191,66 +193,66 @@ def add_kernel(
...
     argv = [sys.executable, "-m", "remote_ikernel"]
 
     # How to connect to kernel
     if interface == "local":
         argv.extend(["--interface", "local"])
         kernel_name.append("local")
-        display_name.append("Local")
+        # display_name.append("Local")
```
Comment (sahil1105) [inline remote_ikernel/manage.py:203]: Let's just remove these lines?

**Review by sahil1105 — APPROVED** [review]
Minor suggestions, but otherwise looks good to me. Thanks @bodo-sai!
Please update the title accordingly.

File: remote_ikernel/manage.py (hunk 1 above)
Comment (bodo-sai) [reply remote_ikernel/manage.py:196]: I think so.

**Pull request merged**